frontend/document/documentregistry.lua

```diff
@@ -160,6 +160,15 @@ function DocumentRegistry:getExtensions()
     return t
 end
 
+--- Get mapping of openable file extensions to providers
+-- @treturn table mapping openable file extensions to a list of providers
+function DocumentRegistry:getOpenableExtensions()
+    local t = self:getExtensions()
+    t["sh"] = nil
```

Comment:
You can just write t.sh

Reply:
I'm slightly in favor of keeping strings in cases like this in lua, because it makes it obvious it's about the string content in the table, and not something that is an object field. While semantics are the same, the syntax distinction indicates intent.

Reply:
Hm, alright.